src/Ng/Jobs/Db2CatalogJob.cs

```diff
@@ -252,42 +270,12 @@ protected override async Task RunInternalAsync(CancellationToken cancellationTok
         }
 
         // Wrapper function for CatalogUtility.CreateHttpClient
-        // Overriden by NgTests.TestableFeed2CatalogJob
+        // Overriden by NgTests.TestableDb2CatalogJob
```

Comment:
Use XML docs so you can point to the actual type.

Reply:
This is an old comment. Also, the jobs don't reference the test projects ;)